active_directory_ldap/help.md

```diff
@@ -352,6 +351,7 @@ paired `\(\)` are supported
 
 # Version History
 
+* 3.2.8 - Hub styling update
```

Comment:
We don't need to bump this since last line is Hub related which didn't get released